### ISSUE TITLE
No email sent

### ISSUE THREAD

**Issue by firmanhadi** [lutraconsulting/mergin#18]
Dear all,

I have set up my server for Mergin. It works well except for the email part.
I have installed and configured Postfix for sending emails, opened port 587, even configured email forwarding with Postale.io.
Before these configurations, I noticed several errors such as Connection refused.
Currently, there is no error message in the terminal, so I was wondering what I missed.
Thanks.

<img width="1103" alt="Screen Shot 2021-09-22 at 08 52 10" src="https://user-images.githubusercontent.com/7415563/134271433-264d0345-8e60-4284-ae0e-0fbaa6104b71.png">

**Comment by wonder-sk** [comment]
Hi Firman, it is a bit hard to guess what could go wrong based on the information you have provided - and the screenshot does not seem to contain any relevant information. I would suggest to first try with some mail server that is known to work well, rather than configuring your local postfix email server - it could be that the mailserver is not configured correctly and other mail servers simply reject any mail from it.

**Comment by firmanhadi** [comment]
Hi Martin, thank you for the response. I will look further based on your suggestion.

**Comment by firmanhadi** [comment]
HI Martin,

I have solved the email problem. Since I use AWS EC2, I had to configure Amazon SES for sending emails. 
The configuration in docker-compose.yml has to be according to the credentials used for Amazon SES.

Thanks.

Firman.
<img width="1123" alt="Screen Shot 2021-09-24 at 13 36 06" src="https://user-images.githubusercontent.com/7415563/134629685-10e34a28-ecf0-4b37-a24c-d18a1dab4edd.png">
